scalajslib/src/mill/scalajslib/ScalaJSModule.scala

```diff
@@ -17,7 +17,7 @@ trait ScalaJSModule extends scalalib.ScalaModule { outer =>
   trait Tests extends TestScalaJSModule {
     override def scalaVersion = outer.scalaVersion()
     override def scalaJSVersion = outer.scalaJSVersion()
-    override def moduleDeps = Seq(outer)
+    override def moduleDeps = outer +: outer.moduleDeps
```

Comment:
We need this, cause otherwise we don't have `sjsir` files of dependent modules in tests

Reply:
Why don't we have the `sjsir` files of dependent modules though? Shouldn't they all be aggregated as part of the `runClasspath`?

Reply:
Hm, yt turns out, that we don't use sjsir files from `runClasspath` in link:
https://github.com/lihaoyi/mill/blob/8a3af216f7ada18d25550b4d4c171d082050bc85/scalajslib/src/mill/scalajslib/ScalaJSModule.scala#L121

We only use libraries(jars) from runtime classpath, whereas sjsir files come from `input`

Reply:
@lihaoyi Do you think we should take sjsir files from runClasspath instead of what I did here? Now I think it's better soultion